pylint/checkers/refactoring.py

```diff
@@ -489,6 +540,36 @@ def _seq_based_ternary_params(node):
         condition = node.slice.value
         return condition, true_value, false_value
 
+    def visit_functiondef(self, node):
+        self._return_nodes[node.name] = []
+        return_nodes = node.nodes_of_class(astroid.Return)
+        self._return_nodes[node.name] = [_rnode for _rnode in return_nodes
+                                         if _rnode.frame() == node.frame()]
+
+    def _check_consistent_returns(self, node): # pylint: disable=redundant-returns-doc
```

Comment:
This is raising a `redundant-returns-doc` because you have a returns section in a function that doesn't return anything. Remove the returns section and then you will be able to remove this disable.